packages/liveblocks-react/src/umbrella-store.ts

```diff
@@ -1209,6 +1208,24 @@ export class UmbrellaStore<M extends BaseMetadata> {
       }
     );
 
+    const channelNotificationSettings: LoadableResource<ChannelsNotificationSettingsAsyncResult> =
+      {
+        signal: DerivedSignal.from(
+          (): ChannelsNotificationSettingsAsyncResult => {
+            const asyncResult = this.#channelsNotificationSettings.get();
+            if (asyncResult.isLoading || asyncResult.error) {
+              return asyncResult;
+            }
```

Comment:
Tiny nit. I renamed `asyncResult` → `result` elsewhere, you may want to do the same rename here for consistency.